### ISSUE TITLE
Error when initializing plugin.

### ISSUE THREAD

**Issue by Codeize** [devtomio/sapphire-plugin-botlist#7]
Client init:
```ts
const client = new SapphireClient({
	defaultPrefix: '!',
	regexPrefix: /^(hey +)?bot[,! ]/i,
	caseInsensitiveCommands: true,
	logger: {
		level: LogLevel.Debug
	},
	shards: 'auto',
	intents: [
		'GUILDS',
		'GUILD_MEMBERS',
		'GUILD_BANS',
		'GUILD_EMOJIS_AND_STICKERS',
		'GUILD_VOICE_STATES',
		'GUILD_MESSAGES',
		'GUILD_MESSAGE_REACTIONS',
		'DIRECT_MESSAGES',
		'DIRECT_MESSAGE_REACTIONS'
	],
	statcord: {
		client_id: '',
		key: `${process.env.STATCORD_KEY}`,
		autopost: true,
		baseUrl: 'https://api.statcord.com/v3',
		debug: true,
		sharding: false
	},
	botList: {
		clientId: '',
		debug: true,
		autoPost: {
			enabled: true,
			interval: 3_600_000,
		},
		keys: {
			topGG: 'YOUR_AWESOME_TOP_GG_API_KEY'
		}
	}
});
````

Error:
```ts
Type '{ clientId: string; debug: true; autoPost: { enabled: true; interval: number; }; keys: { topGG: string; }; }' is not assignable to type 'BotList'.
  Object literal may only specify known properties, and 'debug' does not exist in type 'BotList'.ts(2322)
[index.d.ts(6, 9): ]()The expected type comes from property 'botList' which is declared here on type 'ClientOptions'
```

Any help would be appreciated! 😄

**Comment by devtomio** [comment]
I'll look into this.

**Comment by devtomio** [comment]
Fixed in [`87aff0f`](https://github.com/devtomio/sapphire-plugin-botlist/commit/87aff0f0b932b492dc165e3e1c997bccbfe462b3)

cc: @Codeize

**Comment by devtomio** [comment]
Published [`1.1.0`](https://www.npmjs.com/package/@devtomio/plugin-botlist/v/1.1.0)

**Comment by Codeize** [comment]
Appreciate it!
